debug_toolbar/static/debug_toolbar/js/history.js

```diff
@@ -104,3 +104,5 @@ $$.on(djDebug, "click", ".refreshHistory", function (event) {
     event.preventDefault();
     refreshHistory();
 });
+// As we don't refresh the whole toolbar each fetch or ajax request we need to refresh the history when we open the panel
```

Comment:
```suggestion
// We don't refresh the whole toolbar each fetch or ajax request,
// so we need to refresh the history when we open the panel
```
Small grammar change and updating the line breaks.